src/lib/study-webidl.js

```diff
@@ -285,6 +344,87 @@ async function studyWebIdl(edResults, curatedResults) {
     }
   }
 
+  function checkMembers(target, source) {
+    const selfCheck = !source;
+    source = source ?? target;
+    const knownDuplicates = [];
+    if (!target.idl.members || !source.idl.members) {
+      return;
+    }
+    for (const targetMember of target.idl.members) {
+      if (!targetMember.name) {
+        continue;
+      }
+
+      for (const sourceMember of source.idl.members) {
```

Comment:
I try to be consistent in the loops that are used: if outer loop is a `for`, I actually find it clearer not to mix with a `filter` and `forEach`. I suppose clarity is subjective ;) I'll rewrite.

I note I don't understand why this would be any faster: the call to `filter` would parse the entire list as well, same as the `for` loop. If anything, it will make things slower, since code will filter the list first, then go over the filtered list (`filter` followed by a `for`). But well, given the expected sizes of the lists at hand, any optimization would be overkill in any case.

Reply:
I don't mind keeping it as is, the consistency argument wfm